tests/models/test_transformers.py

```diff
@@ -181,10 +181,13 @@ def test_classify(
     import torch
     from transformers import AutoModelForSequenceClassification
 
+    # Seem to use slightly more memory after torch.compile in 2.8
+    # only for this model jason9693/Qwen2.5-1.5B-apeach
     with vllm_runner(model,
                      max_model_len=512,
                      dtype=dtype,
-                     model_impl="transformers") as vllm_model:
+                     model_impl="transformers",
+                     gpu_memory_utilization=0.85) as vllm_model:
```

Comment:
I start to see a slight memory increase for `jason9693/Qwen2.5-1.5B-apeach` in 2.8 torch.compile, i.e. https://buildkite.com/vllm/ci/builds/28398#0198e856-04da-4929-bbe0-ae0d25ad2ae8/214-4109.  This must have come from change in the past 4 days between https://github.com/huydhn/vllm/commit/9b0187003e62bdb7311b23b5b5026ea8e4e207d3 and https://github.com/huydhn/vllm/commit/24d0c9e6edc4299f62053ee0cb0154ce86b08cb8 because nothing else has changed.

cc @zou3519 if you could spot anything here

Reply:
Oh well, lowering gpu_memory_utilization doesn't seem to help

Reply:
How did you diagnose the problem to torch.compile? Was it using `--enforce-eager`?

Reply:
cc @BoyuanFeng it sounds like Inductor generated a triton kernel that doesn't work on L4 gpus because the machine type doesn't have enough sram

Reply:
Confirm that https://github.com/vllm-project/vllm/pull/23853 fixes the issue, I will wait for that to land given that it has been approved